packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift

```diff
@@ -50,13 +57,13 @@ public class SharedPreferencesPlugin: NSObject, FlutterPlugin, UserDefaultsApi {
   }
 }
 
-/// Returns all preferences stored by this plugin.
-private func getAllPrefs() -> [String: Any] {
+/// Returns all preferences stored by this plugin with speicified prefix.
```

Comment:
Typo: specified

Reply:
no, the preferences are becoming their own species. 

Still spelled wrong for that too, darn.